production/helm/loki/Chart.yaml

```diff
@@ -2,8 +2,8 @@ apiVersion: v2
 name: loki
 description: Helm chart for Grafana Loki in simple, scalable mode
 type: application
-appVersion: 2.8.2
-version: 5.8.0
+appVersion: 2.8.4
```

Comment:
This is probably a mistake. Latest Loki version is 2.8.2

Reply:
Yup. I failed to type numbers into an IDE :(